indico/modules/events/surveys/controllers/management.py

```diff
@@ -164,6 +166,7 @@ class RHOpenSurvey(RHManageSurvey):
     def _process(self):
         if self.survey.state == SurveyState.finished:
             self.survey.end_dt = None
+            self.survey.start_notification_sent = False
```

Comment:
And I guess we don't even show the close button when the survey is in the notready state. So yeah, no need to move anything.